server/manifest.js

```diff
@@ -51,7 +51,7 @@ const manifest = module.exports = {
 
 };
 
-if (process.env.NODE_ENV === 'dev') {
+if (process.env.NODE_ENV !== 'prod') {
```

Comment:
I'll update, because standard is better than better, but abbreviating dev and spelling out production makes my soul hurt.